src/lib/TypeWriter.svelte

```diff
@@ -39,10 +39,14 @@
 	let caret: HTMLSpanElement
 	let textDisplayed = $state(' ')
 	let timeout: ReturnType<typeof setTimeout>
+	let blinking: Animation
 
 	$effect(() => {
-		typewriter(texts, repeat)
-		return () => clearTimeout(timeout)
+		typewriter(texts, repeat).catch((_) => {})
```

Comment:
I'd add this handling function to make future debugging easier if something else comes up
```suggestion
typewriter(texts, repeat).catch((e: unknown) => {
	if (e instanceof DOMException && e.name === 'AbortError') return
	console.error(e)
})
```